bin/benchmark.js

```diff
@@ -126,8 +130,8 @@ for (const manager of argv.managers) {
 const slugs = argv.managers.map(utils.slug)
 
 const hyperfine = spawn('hyperfine', [
-  '--ignore-failure',
```

Comment:
I removed this since from what I could tell, a benchmark should never exit uncleanly unless something unexpected happens. This also makes it possible to test by running a benchmark and making sure it exits cleanly.